dnf5daemon-server/dbus/interfaces/org.rpm.dnf.v0.comps.Group.xml

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN"
+"http://www.freedesktop.org/standards/dbus/1.0/introspect.dtd">
+
+<!--
+Copyright Contributors to the libdnf project.
+
+This file is part of libdnf: https://github.com/rpm-software-management/libdnf/
+
+Libdnf is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 2.1 of the License, or
+(at your option) any later version.
+
+Libdnf is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
+-->
+
+<node>
+<!-- org.rpm.dnf.v0.comps.Group:
+   @short_description: Interface to comps groups
+-->
+<interface name="org.rpm.dnf.v0.comps.Group">
+    <!--
+        list:
+        @options: an array of key/value pairs
+        @groups: array of returned groups with requested attributes
+
+        Get list of groups that match to given filters.
+
+        Following options and filters are supported:
+
+            - attributes: list of strings
+                list of group attributes that are returned
+            - group_with_id: bool (default true)
+                match patterns against ids of the groups
+            - group_with_name: bool (default false)
+                match patterns against names of the groups
```

Comment:
I am a bit puzzled by the `group_with_id` and `group_with_name` option names, as I wouldn't intuitively know what they mean. The only thing similar I found were options for rpm filtering (`with_src`, `with_provides`, `with_filenames`...), but then there are similar ones for advisories that mean something different (`with_bz`, `with_cve`). Plus the `group_` prefix could be seen as "related to groups" or as a verb. What was the intention behind the naming?

Reply:
Thanks for the feedback!
The option names originate in `ResolveSpecSettings` which has the same attributes. But thinking about it again... you are right that the names in this context are puzzling.
How about renaming `group_with_id` -> `match_group_id`, `group_with_name`->`match_group_name` ? If you have another/better idea, it'd be appreciated.

Reply:
`match_group_id` and `match_group_name` sound great, I don't have anything better.